pkg/swap/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.5
+
+# swap command - only minimal Alpine install
+
+# use util-linux to get swapfile utils
+#   at some point, hopefully use our own mkswap and swapon
+RUN apk add --update util-linux
+
+# add the entrypoint and make it executable
+COPY . ./
+RUN chmod +x swap.sh
```

Comment:
I did, this is more of a safety thing. Too many years seeing too many mistakes leaves me happy with a no-cost additional safety net. You really care?